server/i18n_test.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mattermost/mattermost-server/plugin/plugintest"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTranslationsPreInit(t *testing.T) {
```

Comment:
thank you for adding test coverage on this.  I learned some things from your style.